### ISSUE TITLE
Add option to sort composer packages in composer.json for all packages

### ISSUE THREAD

**Issue by samdark** [yiisoft/yii-dev-tool#41]
```json
"config": {
    "sort-packages": true
},
```

Would make composer sort `require` and `require-dev` alphabetically. Would result in less configs and more readable dependencies list.

**Comment by rugabarbo** [comment]
1. Merge #49 
2. [Update yii-dev-tool](https://github.com/yiisoft/yii-dev-tool#upgrade) locally
3. Run `./yii-dev replicate/composer-config`
4. Commit changes of composer.json files in the all repositories

In the future, you can add more common settings to `yii-dev-tool/config/replicate/composer.json` and merge them in the all packages with just one command.

**Comment by rugabarbo** [comment]
I have sent additional fix #50. It is necessary for the correct merging of complex configs. For example: 
```json
{
    "keywords": [
        "yii"
    ],
    "homepage": "http://www.yiiframework.com",
    "authors": [
        {
            "name": "Alexander Makarov",
            "email": "sam@rmcreative.ru",
            "homepage": "http://rmcreative.ru/",
            "role": "Core framework development"
        }
    ],
    "support": {
        "forum": "http://www.yiiframework.com/forum/",
        "wiki": "http://www.yiiframework.com/wiki/",
        "irc": "irc://irc.freenode.net/yii"
    },
    "require": {
        "php": "^7.4"
    },
    "extra": {
        "branch-alias": {
            "dev-master": "3.0.x-dev"
        }
    },
    "config": {
        "sort-packages": true
    }
}
```

**Comment by samdark** [comment]
Worked flawlessly. All packages are updated now.
